pkgs/tools/networking/hp2p/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, openmpi, python2, python2Packages, autoconf, automake } :
```

```suggestion
{ stdenv, fetchFromGitHub, openmpi, python2Packages, autoconf, automake } :
```

You're not using python2 in your package.